src/main/app/src/app/admin/formulier-definities/formulier-definities.component.less

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
Laten we lege files gewoon staan?

Reply:
ja, die laten we altijd staan. maar komt nog wat opmaak in ;)